lib/floe/workflow/payload_template.rb

```diff
@@ -4,31 +4,56 @@ module Floe
   class Workflow
     class PayloadTemplate
       def initialize(payload)
-        @payload = payload
+        @payload = parse_payload(payload)
```

Comment:
I think of `payload` as the unparsed input.
(I also kinda of want to get rid of all unparsed payloads that we store)

could we call this something like `parameters` or `template`?

Reply:
👍 yeah good point